### PR TITLE
Update the people search endpoint to v2

### PR DESCRIPTION
**Pull request by varma77**

### CODE REVIEW

File: lib/api/search.js [hunk 1]
```diff
@@ -42,11 +42,10 @@ module.exports = function search(options, utils) {
             authFlow: options.authFlow,
             baseUrl: options.baseUrl,
             method: 'GET',
-            resource: '/search/profiles',
+            resource: '/search/v2/profiles',
```
Comment (customcommander) [inline lib/api/search.js]: Just checking: the doc says this endpoint is still in alpha meaning (still according to the doc) that it won't be available in production. Shouldn't this endpoint be promoted to beta (with support of a development token) at least?

Comment (varma77) [reply lib/api/search.js]: Yeah we will hold on this.

**Comment by paulwib** [comment]
Closing this for now, feel free to re-open when the endpoint is released.

**Comment by luujam1** [comment]
@paulwib can we re-open this ticket please

**Pull request closed without merging**